__tests__/app.test.js

```diff
@@ -116,3 +116,56 @@ describe('/api/articles', () => {
 			})
 	})
 })
+
+describe('POST /api/articles/:article_id/comments', () => {
+	test('POST:201 adds a comment for an article', () => {
+		return request(app)
```

Comment:
We want an extra test that check what happens with additional properties in the body that is sent, we expect a status 201 and to ignore the extra properties and to something like {
				username: 'rogersop',
				body: 'Such coding many comments wow',
				apple: 'red'
			}